src/features/client/components/Discovery/index.jsx

```diff
@@ -0,0 +1,54 @@
+import { Box, Heading, Flex, Button } from '@chakra-ui/react';
+import { ChevronLeftIcon, ChevronRightIcon } from '@chakra-ui/icons';
+import { restaurantInspectMock } from '../../mocks/restaurantInspectMock';
+import './index.scss';
+import RestaurantInspectForLanding from '../RestaurantInspectForLanding';
+
+function Discovery() {
+  return (
+    <Box as="section" className="container" position="relative">
```

Comment:
Move the position property to appropriate parent, this container shouldn't have this property unless it's used for this container.